packages/react-events/src/dom/Press.js

```diff
@@ -660,45 +536,22 @@ const pressResponderImpl = {
           }
           state.touchEvent = touchEvent;
         }
-        const pressTarget = state.pressTarget;
 
-        if (pressTarget !== null && !targetIsDocument(pressTarget)) {
-          if (
-            pointerType === 'mouse' &&
-            context.isTargetWithinNode(target, pressTarget)
-          ) {
-            state.isPressWithinResponderRegion = true;
-          } else {
-            // Calculate the responder region we use for deactivation, as the
-            // element dimensions may have changed since activation.
-            updateIsPressWithinResponderRegion(
-              touchEvent || nativeEvent,
+        if (isPressed) {
+          const onPressMove = props.onPressMove;
+
+          if (isFunction(onPressMove)) {
+            dispatchEvent(
```

Comment:
Won't this mean that we fire pressmove even if our pointer might be outside the target?

Reply:
Yep